tools/add_triage_signature.py

```diff
@@ -1471,6 +1471,56 @@ def _process_ticket(self, url, issue_key):
             )
 
 
+class WrongBootOrderSignature(ErrorSignature):
+    """
+    This signature creates a report of hosts which should be manually rebooted from the installation disk.
+    """
+
+    ERROR_PATTERN = "a manual booting from installation disk"
+    EVENT_PATTERN = "please boot the host"
+
+    def __init__(self, jira_client):
+        super().__init__(jira_client, signature_label="wrong_boot_order",
+                         comment_identifying_string="h1. Wrong boot order")
+
+    def _update_ticket(self, url, issue_key, should_update=False):
```

Comment:
parent class method renamed, it's now `_process_ticket`, and it should accept should_update anymore

Reply:
fixed